en/advanced_config/parameter_reference.md

```diff
@@ -12950,6 +12950,13 @@ table {
  <td>1000.0</td>
  <td>m</td>
 </tr>
+<tr>
```

Comment:
Please remove changes to this file. The parameters are updated from PX4-Autopilot automatically - so after your PR merges we'll do a rebase and get this.

Reply:
Actually, I'll do it

Reply:
good to know, thanks!